test/it/uk/gov/hmrc/component/CaseSpec.scala

```diff
@@ -89,6 +95,42 @@ class CaseSpec extends BaseFeatureSpec {
       responseCase.status shouldBe CaseStatus.NEW
     }
 
+    scenario("Extra fields are ignored when creating a case") {
+      When("I create a new case with extra fields")
+      val result: HttpResponse[String] = Http(s"$serviceUrl/cases")
+        .headers(Seq(CONTENT_TYPE -> JSON))
+        .postData(c3Json.toString()).asString
+
+      Then("The response code should be created")
+      result.code shouldEqual CREATED
+
+      And("The case is returned in the JSON response")
+      val responseCase = Json.parse(result.body).as[Case]
+      responseCase.reference shouldBe "1"
+      responseCase.status shouldBe CaseStatus.NEW
+      responseCase.createdDate should not be ZonedDateTime.parse("2000-01-02T00:00:00+01:00[Europe/Paris]")
```

Comment:
I think we have a matcher in the NewCastRequestSpec that lets you specify a rough date you could use for this

Reply:
It ends up something like `responseCase.createdDate shouldBe roughly(ZonedDateTime.now())`

Reply:
Refactored the matcher to live in a util object within the test folder and used it for the above.